lib/Coffer/Path.hs

```diff
@@ -125,6 +126,14 @@ entryPathName = view $ pathSegments . last1 . to unPathSegment
 entryPathParentDir :: Lens' EntryPath Path
 entryPathParentDir = pathSegments . initNE . from pathSegments
 
+-- | Gets all parents paths of given entry path.
```

Comment:
Please add some doctests or tasty tests, whichever you prefer.

Reply:
Added doctest